.abs/main.yaml

```diff
@@ -1,4 +1,4 @@
-chart-dir: helm/grafana-app
+chart-dir: helm/grafana
```

Comment:
If you don't rename the directory in the helm directory, this shouldn't be renamed either